civihr_default_theme/scss/base/_variables.scss

```diff
@@ -28,6 +28,7 @@ $crm-gray-darker-alt: #404454 !default; // page title
 $crm-grizzly:         #dcdddd !default;
 $crm-color-neutral:   #949caa !default; // gray shade
 $crm-grayish-blue:    #f4f7f8 !default;
+$crm-gray-matte:      $crm-color-neutral; // disabled font
```

Comment:
Variable is assigned with another variable? Why we don't use just `$crm-color-neutral`?

Reply:
`$crm-gray-matte` is used in a partial `/org.civicrm.shoreditch/scss/civicrm/common/_datepickers.scss"`. This partial comes form shoreditch. And we don't have `$crm-gray-matte` variable in SSP theme.

I tried to find the variation of `gray-*`  color appropriate for disabled date, but non were suitable except the value for `$crm-color-neutral`,  so, the `$crm-color-neutral$` is assigned to '$crm-gray-matte`, avoiding to change the variable name in above partial.